services/pull/merge.go

```diff
@@ -39,19 +39,19 @@ import (
 )
 
 // GetDefaultMergeMessage returns default message used when merging pull request
-func GetDefaultMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issues_model.PullRequest, mergeStyle repo_model.MergeStyle) (string, error) {
+func GetDefaultMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issues_model.PullRequest, mergeStyle repo_model.MergeStyle) (string, string, error) {
```

Comment:
I recommend named returns in this case, simply for clarification.

Reply:
I'm not sure.

See Go's [Code Review comments](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters):

> Naked returns are okay if the function is a handful of lines. Once it's a medium sized function, be explicit with your return values. Corollary: it's not worth it to name result parameters just because it enables you to use naked returns. Clarity of docs is always more important than saving a line or two in your function.

Reply:
Yes, and in this case I don't want you to use them for brevity, but to clarify the role of the two string returns values.
That is exactly how I read the above sentence:
> If it makes more sense to have named returns, use them. Otherwise don't.

This is one of the cases where I think it applies.

Reply:
Because otherwise you have to guess what the individual parameters could mean.

Reply:
Yes, I see.